typescript/db/src/repository/image-processing.ts

```diff
@@ -96,10 +120,12 @@ export const processImage: Repository["imageProcessing"]["processImage"] =
       thumbnail100Url,
       thumbnail200Url,
       thumbnail500Url,
-    },
-    getImage,
-    putImage
-  ) => {
+    } = params;
+    if (shouldNotLoadImage(params)) {
+      return params as AsyncReturnType<
```

Comment:
I'm curious as to why it is necessary and makes it work, and why it doesn't work without

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions